README.rst

```diff
@@ -0,0 +1,38 @@
+############################
+ts_positionmeasurementdevice
+############################
+
+PositionMeasurementDevice is a Commandable SAL Component for the `Vera C. Rubin Observatory <https://lsst.org>`_.
+It controls devices that measure relative position displacement over a period of time.
```

Comment:
Is "relative" important in this context? I realize micrometers are relative, but I can imagine other devices that might not be.